pulsar-broker/src/main/java/org/apache/pulsar/broker/service/PublishRateLimiterDisable.java

```diff
@@ -59,7 +59,7 @@ public void update(PublishRate maxPublishRate) {
     @Override
     public boolean tryAcquire(int numbers, long bytes) {
         // No-op
-        return false;
+        return true;
```

Comment:
This problem has been reported as  #10603 and it's fixed by PR #11442.

Reply:
I'm well aware this was fixed by another PR, but in case my PR goes through before my fixes won't work unless this is set to true.

Reply:
yes that's fine.

Reply:
Resolved.